actions/login.php

```diff
@@ -38,23 +37,20 @@
 
 $result = elgg_authenticate($username, $password);
 if ($result !== true) {
-	register_error($result);
-	forward(REFERER);
+	return elgg_error_response($result, REFERRER, ELGG_HTTP_FORBIDDEN);
```

Comment:
If we give the user a UI element to initiate the request, should we ever return a 4xx error?